js/scroll.js

```diff
@@ -1,18 +1,47 @@
-const SCROLL_DELAY = 3000;
-var scrollDistance;
-var intervalScroll;
+const SCROLL_PLAN_DELAY = 3000, SCROLL_INFO_DELAY = 3000;
+var intervalScrollPlan, intervalScrollInfo;
+var infoContainer = document.getElementById('info-container');
```

Comment:
This works, when the document is completely loaded.
Usually, the scripts are loaded before the document.
My workaround would be to create a function.

```
function getInfoContainer() {
    return 
}
```

Global functions and global variables are reachable from every JavaScript file.